### PR TITLE
Improvement translation in zh-Hant

### PR DESCRIPTION
**Pull request by X-EROS**
# SCP:SL Translation PR Template

## Language Name
zh-Hant

## State Type of change (New language, changed existing Etc.)
Fixed others from making trouble

# Checklist:
- [ ] I have tested the translation changes ingame
- [x] I have made sure to not commit changes to multiple translations at the same time, or non-translation repo files
- [x] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  (Exception to this is when edits has __only__ been made in-browser. If this is the case, also check this box.)
- [x] I have made sure that I'm **not** submitting a duplicate translation 
- [x] I have made sure to **not** included any advertisements/pointers to non-game stuff other than the translation author in the credits file
- [x] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of **new** translations (Can be tested [Here](https://rextester.com/WDXPS97501)). Also check this box if the change is **not** a new translation

### CODE REVIEW

**Comment by X-EROS** [comment]
Badges.txt from zh-Hans to zh-Hant 這種行為是不是被允許

File: Translations/credits.txt [hunk 1]
```diff
@@ -28,4 +28,4 @@ th by AGKK Histoire & DatPanda
 tr by Jhrino, MHS, vexdy, Gonsez
 uk by TRI99ER
 zh-Hans by Schrodinger_Sig, ShingekiNoRex, TitanSix & Tychus_Xiaoyi
-zh-Hant by X-EROS & Arkira-Kouyo
```
Comment (MichalPetryka) [inline Translations/credits.txt:31]: Please don't remove others from credits

Comment (X-EROS) [reply Translations/credits.txt:31]: He stole a translation from zh-Hans,  is that behavior be allow?

File: Translations/zh-Hant/Badges.txt [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
-SCP:SL 全球審核員
-SCP:SL 首席遊戲測試
-SCP:SL 首席遊戲邏輯程序員
-SCP:SL 遊戲關卡設計師
+SCP:SL 藝術主管
+SCP:SL 藝術家
+SCP:SL 後端程式設計師
+SCP:SL 後端程序設計經理
+SCP:SL 開發人員
+SCP:SL 活動得獎者
```
[the remark this answers is not in the document]
Comment (X-EROS) [reply Translations/zh-Hant/Badges.txt:25]: 按照你的說法，直接忽視台港叫法，什麼叫standard names in business?這些商業用的職銜是根據我自己在香港的經驗而得出來，而且為了確保字眼是台灣人也明白，也找來了在steam社群翻譯的台灣人幫忙寫出他們的看法，在business上例如CEO : "行政總裁（香港和東南亞的稱呼）是在一個企業中負責日常經營管理的最高級管理人員。是CEO（Chief Executive Officer）最恰當的翻譯，不過CEO（Chief Executive Officer）在中國被逐字生硬翻譯成“首席執行官”。但由於“首席執行官”這個名詞在中國內地已經廣泛傳開，人們已經慢慢習慣了這個不恰當的稱謂。"。兩地翻譯原本就不同，在地化翻譯令層次可提高不少，例如:「視頻」在台灣/香港稱為「影片」，若果簡轉繁也叫做翻譯，以後簡中直接繁化使用就可以，反正lorry and truck is same thing。

[the remark this answers is not in the document]
Comment (X-EROS) [reply Translations/zh-Hant/Badges.txt:25]: Badges.txt中除了部分幾個看似是自己翻譯，其餘都跟簡中是一樣，，當初我去找電腦相職的職位，根本找不到繁中的"全棧程序員"，當我問了相關人士，原來前+後=全端，全棧是大陸用詞，這樣也得出一個結論，#428是帶有破壞性的翻譯，破壞了原先在地化的翻譯，而官方也同意了直接簡變繁翻譯的方法，因此，除非官方能夠說服我或者approve這次request，否則之後的翻譯均使用簡轉繁的方法。

[the remark this answers is not in the document]
Comment (Tychus-Xiaoyi) [reply Translations/zh-Hant/Badges.txt:25]: 您好，正如之前所提及的，本地化語言是由所有貢獻者一起完成的，#428早於您發佈，我們需要測試語言在遊戲中的可用性，您在此之前並沒有提供文本所以我採納了。文本是可以在遊戲正式版本發佈前隨時更改的，可能您對我的說法有所誤解，我所指的問題是沒必要將諸如“獲獎者”“得獎者”這樣的用語故意區別開，我也並沒有否定您的翻譯只是提出建議，其它本地化語言翻譯我完全讚同您的，希望您可以理解。

**Comment by ShingekiNoRex** [comment]
Hey, there's nothing wrong about the PR itself, but removing others from credits is not allowed. Even though one makes only one little change, he has the right to be credited. I agree that simply translating from Hans is not appropriate, but he also has committed changes other than that one. We will approve this PR as long as you dont remove him from credits.

**Pull request merged**